docs/validations.stories.tsx

```diff
@@ -118,3 +119,46 @@ export const GroupedRequired: StoryObj = () => {
   );
 };
 GroupedRequired.storyName = "Grouped Required";
+
+export const ValidationRedesign: StoryObj = () => {
+  return (
+    <CarbonProvider validationRedesignOptIn>
+      <Textbox
+        label="Textbox label"
+        inputHint="Hint text"
+        value="value"
+        error="Error Message (Fix is required)"
+      />
+
+      <Textbox
+        label="Textbox label"
+        inputHint="Hint text"
+        value="value"
+        warning="Warning Message (Fix is optional)"
+      />
+
+      <RadioButtonGroup
+        legend="Radio Button Group legend"
+        legendHelp="Legend help"
+        name="name"
+        error="Error Message (Fix is required)"
+        mt={2}
+      >
+        <RadioButton id="radio-1" value="radio1" label="Radio Option 1" />
+        <RadioButton id="radio-2" value="radio2" label="Radio Option 2" />
+      </RadioButtonGroup>
+
+      <RadioButtonGroup
+        legend="Radio Button Group legend"
+        legendHelp="Legend help"
+        name="name"
+        warning="Warning Message (Fix is optional)"
+        mt={2}
+      >
+        <RadioButton id="radio-1" value="radio1" label="Radio Option 1" />
+        <RadioButton id="radio-2" value="radio2" label="Radio Option 2" />
+      </RadioButtonGroup>
```

Comment:
**Issue:** You'll need to give these Radio Buttons unique values otherwise this happens (see screen recording)

https://github.com/Sage/carbon/assets/56251247/38fce3df-ad02-47bc-b44b-c8c0b71c642e

Reply:
Oh good spot, didn't even think to test those